test/unit/node/heart.test.ts

```diff
@@ -38,12 +38,16 @@ describe("Heart", () => {
 
     heart = new Heart(pathToFile, mockIsActive(true))
     heart.beat()
+    // HACK@jsjoeio - beat has some async logic but is not an async method
+    // Therefore, we have to create an artificial wait in order to make sure
+    // all async code has completed before asserting
+    await new Promise((r) => setTimeout(r, 100))
     // Check that the heart wrote to the heartbeatFilePath and overwrote our text
     const fileContentsAfterBeat = await readFile(pathToFile, { encoding: "utf8" })
     expect(fileContentsAfterBeat).not.toBe(text)
     // Make sure the modified timestamp was updated.
     const fileStatusAfterEdit = await stat(pathToFile)
-    expect(fileStatusAfterEdit.mtimeMs).toBeGreaterThan(fileStatusBeforeEdit.mtimeMs)
+    expect(fileStatusAfterEdit.mtimeMs).toBeGreaterThanOrEqual(fileStatusBeforeEdit.mtimeMs)
```

Comment:
It occurs to me that if we allow it to be equal then technically we are not testing anything because this will pass if the file is not modified.  Any idea if this still flakes if we remove the equal?

Reply:
Maybe what we can do is set the modified time explicitly before the heartbeat and then do the comparison?

https://nodejs.org/api/fs.html#filehandleutimesatime-mtime

Reply:
Oh good point. Let me remove the Equal and check. I also like the idea of explicitly setting. That gives us more control.